{
  "uid" : "7e58dcb1976bea9b",
  "name" : "Test submenu and button on the Careers page",
  "fullName" : "Test Redirection of Pages: Test submenu and button on the Careers page",
  "historyId" : "f532336fa6cdc1e7248f1e97b0059225",
  "time" : {
    "start" : 1698820921178,
    "stop" : 1698820943845,
    "duration" : 22667
  },
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"/html/body/div[1]/div[3]/div/div/div/div/main/article/div/section/div[4]/div/div/div/div[2]/div/h2/span/a\"}\n  (Session info: chrome=114.0.5735.133); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x55d7ccbf14e3 <unknown>\n#1 0x55d7cc920c76 <unknown>\n#2 0x55d7cc95cc96 <unknown>\n#3 0x55d7cc95cdc1 <unknown>\n#4 0x55d7cc9967f4 <unknown>\n#5 0x55d7cc97c03d <unknown>\n#6 0x55d7cc99430e <unknown>\n#7 0x55d7cc97bde3 <unknown>\n#8 0x55d7cc9512dd <unknown>\n#9 0x55d7cc95234e <unknown>\n#10 0x55d7ccbb13e4 <unknown>\n#11 0x55d7ccbb53d7 <unknown>\n#12 0x55d7ccbbfb20 <unknown>\n#13 0x55d7ccbb6023 <unknown>\n#14 0x55d7ccb841aa <unknown>\n#15 0x55d7ccbda6b8 <unknown>\n#16 0x55d7ccbda847 <unknown>\n#17 0x55d7ccbea243 <unknown>\n#18 0x7f84a1290402 <unknown>\n\n",
  "statusTrace" : "  File \"/usr/local/lib/python3.10/dist-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/usr/local/lib/python3.10/dist-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"steps/PageRedirectionSteps.py\", line 103, in step_impl\n    button_element = context.driver.find_element(By.XPATH, xpath)\n  File \"/home/addweb/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 740, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n  File \"/home/addweb/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 346, in execute\n    self.error_handler.check_response(response)\n  File \"/home/addweb/.local/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py\", line 245, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "testStage" : {
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"/html/body/div[1]/div[3]/div/div/div/div/main/article/div/section/div[4]/div/div/div/div[2]/div/h2/span/a\"}\n  (Session info: chrome=114.0.5735.133); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x55d7ccbf14e3 <unknown>\n#1 0x55d7cc920c76 <unknown>\n#2 0x55d7cc95cc96 <unknown>\n#3 0x55d7cc95cdc1 <unknown>\n#4 0x55d7cc9967f4 <unknown>\n#5 0x55d7cc97c03d <unknown>\n#6 0x55d7cc99430e <unknown>\n#7 0x55d7cc97bde3 <unknown>\n#8 0x55d7cc9512dd <unknown>\n#9 0x55d7cc95234e <unknown>\n#10 0x55d7ccbb13e4 <unknown>\n#11 0x55d7ccbb53d7 <unknown>\n#12 0x55d7ccbbfb20 <unknown>\n#13 0x55d7ccbb6023 <unknown>\n#14 0x55d7ccb841aa <unknown>\n#15 0x55d7ccbda6b8 <unknown>\n#16 0x55d7ccbda847 <unknown>\n#17 0x55d7ccbea243 <unknown>\n#18 0x7f84a1290402 <unknown>\n\n",
    "statusTrace" : "  File \"/usr/local/lib/python3.10/dist-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/usr/local/lib/python3.10/dist-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"steps/PageRedirectionSteps.py\", line 103, in step_impl\n    button_element = context.driver.find_element(By.XPATH, xpath)\n  File \"/home/addweb/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 740, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n  File \"/home/addweb/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 346, in execute\n    self.error_handler.check_response(response)\n  File \"/home/addweb/.local/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py\", line 245, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
    "steps" : [ {
      "name" : "Given The user is on the \"stg\" site",
      "time" : {
        "start" : 1698820922181,
        "stop" : 1698820936111,
        "duration" : 13930
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "When The user hovers over the \"company\" menu",
      "time" : {
        "start" : 1698820936111,
        "stop" : 1698820936459,
        "duration" : 348
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "And The user clicks on the \"careers\" submenu link and verify the redirection \"careers\" page",
      "time" : {
        "start" : 1698820936459,
        "stop" : 1698820938621,
        "duration" : 2162
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "When The user clicks on the \"linkedin_page_jobs_2\" button",
      "time" : {
        "start" : 1698820938621,
        "stop" : 1698820943775,
        "duration" : 5154
      },
      "status" : "broken",
      "statusMessage" : "selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"/html/body/div[1]/div[3]/div/div/div/div/main/article/div/section/div[4]/div/div/div/div[2]/div/h2/span/a\"}\n  (Session info: chrome=114.0.5735.133); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x55d7ccbf14e3 <unknown>\n#1 0x55d7cc920c76 <unknown>\n#2 0x55d7cc95cc96 <unknown>\n#3 0x55d7cc95cdc1 <unknown>\n#4 0x55d7cc9967f4 <unknown>\n#5 0x55d7cc97c03d <unknown>\n#6 0x55d7cc99430e <unknown>\n#7 0x55d7cc97bde3 <unknown>\n#8 0x55d7cc9512dd <unknown>\n#9 0x55d7cc95234e <unknown>\n#10 0x55d7ccbb13e4 <unknown>\n#11 0x55d7ccbb53d7 <unknown>\n#12 0x55d7ccbbfb20 <unknown>\n#13 0x55d7ccbb6023 <unknown>\n#14 0x55d7ccb841aa <unknown>\n#15 0x55d7ccbda6b8 <unknown>\n#16 0x55d7ccbda847 <unknown>\n#17 0x55d7ccbea243 <unknown>\n#18 0x7f84a1290402 <unknown>\n\n",
      "statusTrace" : "  File \"/usr/local/lib/python3.10/dist-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/usr/local/lib/python3.10/dist-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"steps/PageRedirectionSteps.py\", line 103, in step_impl\n    button_element = context.driver.find_element(By.XPATH, xpath)\n  File \"/home/addweb/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 740, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n  File \"/home/addweb/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 346, in execute\n    self.error_handler.check_response(response)\n  File \"/home/addweb/.local/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py\", line 245, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
      "steps" : [ ],
      "attachments" : [ {
        "uid" : "f51554b4fa38c57c",
        "name" : "Error Message",
        "source" : "f51554b4fa38c57c.txt",
        "type" : "text/plain",
        "size" : 948
      }, {
        "uid" : "cdc6921eb34fda0f",
        "name" : "Screenshot of https://stg.voltactivedata.com/company/careers/ Page",
        "source" : "cdc6921eb34fda0f.png",
        "type" : "image/png",
        "size" : 93981
      } ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 2,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    }, {
      "name" : "Then The user should be redirected to the \"linkedin_page_jobs\" page",
      "time" : {
        "start" : 1698820943845,
        "stop" : 1698820943845,
        "duration" : 0
      },
      "status" : "skipped",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 5,
    "attachmentsCount" : 2,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "severity",
    "value" : "normal"
  }, {
    "name" : "feature",
    "value" : "Test Redirection of Pages"
  }, {
    "name" : "framework",
    "value" : "behave"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ ]
  },
  "source" : "7e58dcb1976bea9b.json",
  "parameterValues" : [ ]
}